docs/HowToFAQ/FAQNameConflict.txt

```diff
@@ -0,0 +1,2 @@
+/Users/davidcok/projects/dafny/dafny/docs/HowToFAQ/FAQNameConflict.dfy(6,23): Error: Wrong number of type arguments (1 instead of 0) passed to module: Result
```

Comment:
Maybe use a version of this file generated by running `dafny /useBaseNameForFileName`?

Reply:
That would just be extra explaining to do in the answer... and not pertinent to the question, I think